.github/workflows/staging.yml

```diff
@@ -70,19 +84,92 @@ jobs:
 
       - name: Service with ingress
         run: |
-          echo "ENVIRONMENT_URL=https://${{ matrix.service }}.kesaseteli.${{ env.BASE_DOMAIN }}" >> $GITHUB_ENV
+          echo "ENVIRONMENT_URL=https://kesaseteli-${{ matrix.service }}-${{ env.BASE_DOMAIN }}" >> $GITHUB_ENV
```

Comment:
I think dot is preferred before base-domain. With hyphen DNS probably goes to wrong IP.

Reply:
fixed